src/modules/launcher/Wox.Infrastructure/Image/ImageLoader.cs

```diff
@@ -184,9 +184,13 @@ public static ImageSource Load(string path, bool loadFullImage = false)
                 string hash = EnableImageHash ? _hashGenerator.GetHashFromImage(img) : null;
                 if (hash != null)
                 {
+                    int ImageCacheValue;
                     if (GuidToKey.TryGetValue(hash, out string key))
                     { // image already exists
-                        img = ImageCache[key];
+                        if(ImageCache.Usage.TryGetValue(path, out ImageCacheValue))
```

Comment:
[nit] Earlier this was only checking whether GuidToKey contained the hash and if so it would retrieve the image. Why is a change required here for checking the ImageCache as well?

Reply:
Due to resizing of the dictionary, there are cases when the key is not a part of the imageCache. That's why this has been added to check if the img exists even after the resizing. Without this condition, it throws an exception while trying to retrieve that item.